src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
+  let result;
+
+  if (operation === '+') {
+    result = num1.plus(num2).toString();
+  }
+  if (operation === '-') {
+    result = num1.minus(num2).toString();
+  }
+  if (operation === 'x') {
+    result = num1.times(num2).toString();
+  }
+  if (operation === '÷') {
+    result = num1.div(num2).toString();
+  }
+  if (operation === '%') {
+    result = num1.mod(num2).toString();
+  }
```

Comment:
The `%` modifier should  turn the number into a percentage based number, `e.g. 95 % == .95` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))